### PR TITLE
Merge/docs

### PR DESCRIPTION
**Pull request by astrit**
This PR merges docs ( seb.io ) into main Green repo. 
Also sets the workflows to publish on staging and production.

### CODE REVIEW

File: apps/docs/public/fonts/SEBSansSerif-Bold.woff
Comment (splashdust) [inline apps/docs/public/fonts/SEBSansSerif-Bold.woff:1]: The font files should come from the fonts package, right? Shouldn't need to bundle them here

File: apps/docs/public/core-out/index.js
Comment (splashdust) [inline apps/docs/public/core-out/index.js:1]: This one should be removed

**Review by splashdust — CHANGES_REQUESTED** [review]

File: apps/docs/public/fonts/SEBSansSerif-Bold.woff
Comment (astrit) [reply apps/docs/public/fonts/SEBSansSerif-Bold.woff:1]: I guess will update this once we move the fonts package in here also ✌️

**Review by splashdust — APPROVED** [review]

**Review by vsjolander — APPROVED** [review]
LGTM!

**Pull request merged**